docs/src/description/convenience-syntax.md

```diff
@@ -0,0 +1,55 @@
+# Convenience syntax
+
+To make writing constraints easier, AirScript provides a number of syntactic conveniences. These are described in this section.
+
+## List Comprehension
+
+List comprehension provides syntactic convenience for creating new vectors. It is similar to the list comprehension syntax in Python. The following examples show how to use List Comprehension in AirScript.
+
+```
+let x = [a * 2 for a in b]
+```
+This will create a new vector with the same length as `b` and each element will be twice the corresponding element in `b`.
```

Comment:
nit: I would probably say "...and the value of each element will be twice that of the corresponding element in `b`."